spec/support/active_admin_request_helpers.rb

```diff
@@ -0,0 +1,27 @@
+require 'action_dispatch'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.